cluster/contrail-control.manifest

```diff
@@ -7,7 +7,7 @@
 	"containers":[{
 	    "name": "contrail-control",
 	    "image": "opencontrail/control:2.20",
-	    "command": ["/bin/bash", "-c", "echo 127.0.0.1 $(hostname) >> /etc/hosts && /usr/bin/contrail-control >>/var/log/contrail/contrail-control.stdout 2>/var/log/contrail/contrail-control.err"],
+	    "command": ["/bin/bash", "-c", "/usr/bin/contrail-control --DISCOVERY.server=127.0.0.1 --DISCOVERY.port=5998 >>/var/log/contrail/contrail-control.stdout 2>/var/log/contrail/contrail-control.err"],
```

Comment:
In all the setups we tried (vagrant and aws), hostname was correctly
resolvable to the container's host's address. Hence we thought, it is
better not to override that.

On Tue, Sep 1, 2015 at 3:33 PM, Pedro Marques <notifications@github.com>
wrote:

> In cluster/contrail-control.manifest
> <https://github.com/Juniper/contrail-kubernetes/pull/40#discussion_r38480505>
> :
>
> > @@ -7,7 +7,7 @@
> >  	"containers":[{
> >  	    "name": "contrail-control",
> >  	    "image": "opencontrail/control:2.20",
> > -	    "command": ["/bin/bash", "-c", "echo 127.0.0.1 $(hostname) >> /etc/hosts && /usr/bin/contrail-control >>/var/log/contrail/contrail-control.stdout 2>/var/log/contrail/contrail-control.err"],
> > +	    "command": ["/bin/bash", "-c", "/usr/bin/contrail-control --DISCOVERY.server=127.0.0.1 --DISCOVERY.port=5998 >>/var/log/contrail/contrail-control.stdout 2>/var/log/contrail/contrail-control.err"],
>
> Is the control node now executing correctly if /etc/hosts doesn't have an
> entry for hostname ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/40/files#r38480505>.
>